src/script/script.h

```diff
@@ -17,7 +17,14 @@
 #include <string>
 #include <vector>
 
-static const unsigned int MAX_SCRIPT_ELEMENT_SIZE = 520; // bytes
+// Maximum number of non-push operations per script
+#define MAX_OPS_PER_SCRIPT 201
```

Comment:
Please don't use defines to declare constants, follow the rest of the code and use `static const <type>`